tech/post/2017-11-10-Kubernetes-Affinity&AntiAffinity.md

```diff
@@ -0,0 +1,374 @@
+# Affinity and AntiAffinity
+
+## 1. User Cases

```

Comment:
Use cases, also need to correct the others below:)

Reply:
Done